custom-metrics-stackdriver-adapter/README.md

```diff
@@ -166,7 +171,16 @@ by a simple name, as defined in [custom metric naming rules].
      stackdriverService.Projects.TimeSeries.Create("projects/<your project ID>", request).Do()
      ```
 
-#### Examples
+### Export external metrics to Stackdriver
```

Comment:
It isn't really about exporting metrics to Stackdriver, rather fetching them from there. Maybe instead of dividing this into external vs custom (which is not entirely correct as @MaciekPytel correctly points out custom metrics are a subset of external metrics) we can do fetching vs exporting to stackdriver.

So one paragraph would be about how to get your metrics from Stackdriver (and link to existing GCP metrics) - these two requirements that you have below and second paragraph is if you want to use custom metrics coming from your Kubernetes cluster, this is how you get them into Stackdriver. WDYT?

Reply:
Good suggestion, reorganized the doc. In the meantime, I managed to drop the assumption that the metric has kind GAUGE as suggested by @MaciekPytel.

Reply:
This is great, thank you! @MaciekPytel any chance we will be able to follow this train of explanation in other docs?

Reply:
:+1: I like how you reorganized the doc. As to your question @bskiba - I'd rather discuss it with you offline.